README.md

```diff
@@ -133,6 +133,13 @@ This project was migrated to GitHub from a private server at v0.8.0. Changes pri
 
 This project follows [semantic versioning](https://semver.org/). This changelog follows the guidelines published on [keepachangelog.com](https://keepachangelog.com/en/1.0.0/).
 
+### Unreleased
+
+#### Added
+
+* `read_bulk()` convenience function to read multiple StarTable files at once into a single `Bundle`. 
+* `import_from_word()` utility function to parse table blocks from tables in Microsoft Word `*.docx` files. 
```

Comment:
I don't think you actually implemented an 'import_from_word' function, no?  I couldn't find it, and a call to 'read_word' was commented out in bulk.py